flask_of_oil/__init__.py

```diff
@@ -0,0 +1,5 @@
+__version__ = '0.0.1'
```

Comment:
not 1.0.0?

Reply:
will go to 1.0.0 when merged, right now i am testing the publishing part and will use 0.0.1,0.0.2 etc..
after publishing, the version cannot be re-published in pypi, just deleted, so we have to make it right